content/applications/general/calendars/google/google_calendar_credentials.rst

```diff
@@ -2,89 +2,130 @@
 
-- Select *Web browser (Javascript)* 
-  as calling source and *User data* as kind of data.
+Now that you have created your API project and enabled the Google Calendar API, you need to create
+credentials. Begin by clicking :guilabel:`Create Credentials`. Google will then guide you through
+four steps to create your API credentials.
 
-.. image:: media/google_calendar_credentials05.png
-    :align: center
+- In the first step, **Credential Type**, select the *Google Calendar API* and *User Data* options.
+  Then, click :guilabel:`Next`.
 
-- Then you can create a Client ID.
-  Enter the name of the application (e.g. Odoo Calendar) and the allowed pages on 
-  which you will be redirected. The *Authorized JavaScript origin* is your 
-  Odoo's instance URL. The *Authorized redirect URI* is your Odoo's instance 
-  URL followed by '/google_account/authentication'.
+.. image:: google_calendar_credentials/credential_type.png
+   :align: center
+   :alt: Select Google Calendar API and User Data for the Credential Type.
 
-.. image:: media/google_calendar_credentials06.png
-    :align: center
+- In the **OAuth Consent Screen** step, type *Odoo* in the :guilabel:`App name` field, select your
+  email address for the :guilabel:`User support email` field, and type your email address for the
+  :guilabel:`Developer contact information` section. Click :guilabel:`Save and Continue`.
 
-- Go through the Consent Screen step by entering a product name 
-  (e.g. Odoo Calendar). Feel free to check the customizations options 
-  but this is not mandatory. The Consent Screen will only show up when you 
-  enter the Client ID in Odoo for the first time.
+- Skip the **Scopes** step by clicking :guilabel:`Save and Continue`.
 
-- Finally you are provided with your **Client ID**. Go to *Credentials* to 
-  get the **Client Secret** as well. Both of them are required in Odoo.
+- In the **OAuth Client ID** step, select *Website application* for the :guilabel:`Application
+  Type` field and type *My Odoo Database* for the :guilabel:`Name`.
+  
+  - In the **Authorized JavaScript Origins** section, click :guilabel:`+ Add URI` and type your
+    company's Odoo URL address.
 
-.. image:: media/google_calendar_credentials07.png
-    :align: center
+  - In the **Authorized redirect URIs** section, click :guilabel:`+ Add URI` and type your
+    company's Odoo URL address followed by */google_account/authentication*. Finally, click
+    :guilabel:`Create` and :guilabel:`Done`.
+
+.. image:: google_calendar_credentials/uri.png
+   :align: center
+   :alt: Add the authorized JavaScript origins and the authorized redirect URIs.
+
+After successfully creating a new API project, enable the Google Calendar API, and creating the
+Google Calendar API credentials, you should now have a Client ID and Client Secret.
+
+Client ID & Client Secret
+-------------------------
+
+The Client ID and the Client Secret are both needed to connect Google Calendar to Odoo. To find the
```

Comment:
Would **bolden** these since they're being referenced and defined for the first time right after the last paragraph's conclusion.

```suggestion
The **Client ID** and the **Client Secret** are both needed to connect Google Calendar to Odoo. To find the
```